tests/src/rules/no-duplicates.js

```diff
@@ -119,49 +119,48 @@ ruleTester.run('no-duplicates', rule, {
 
     test({
       code: "import { x, /* x */ } from './foo'; import {//y\ny//y2\n} from './foo'",
-      output: "import { x, /* x */ //y\ny//y2\n} from './foo'; ",
       errors: ['\'./foo\' imported multiple times.', '\'./foo\' imported multiple times.'],
     }),
 
     test({
       code: "import {x} from './foo'; import {} from './foo'",
-      output: "import {x} from './foo'; ",
+      output: "import { x } from './foo'; ",
       errors: ['\'./foo\' imported multiple times.', '\'./foo\' imported multiple times.'],
     }),
 
     test({
       code: "import {x} from './foo'; import {} from './foo'; import {/*c*/} from './foo'; import {y} from './foo'",
-      output: "import {x/*c*/,y} from './foo';   ",
+      output: "import { x, y } from './foo';  import {/*c*/} from './foo'; ",
```

Comment:
same here; the comment must appear after the x and before the y, or else it's likely not to make sense.